docs/content/how-to/azure/create-azure-cluster-with-options.md

```diff
@@ -1,38 +1,13 @@
 # Create an Azure cluster with Additional Options
 This document describes how to set up an Azure cluster with Hypershift with additional flag options. 
 
-Creating an Azure cluster with Hypershift without any additional flag options can be found [here](create-azure-cluster.md).
+Creating an Azure cluster with Hypershift without any additional flag options can be found [here](create-azure-cluster_on_aks.md).
 
 ## Prerequisites
-See the Prerequisites section in [Create an Azure Cluster](./create-azure-cluster.md#prerequisites)
-
-## Creating the Cluster in an Existing Resource Group
```

Comment:
This is already demonstrated in the main AKS doc so I removed it from here rather than duplicating the same command.